RELEASES.md

```diff
@@ -1,6 +1,8 @@
 # Release Notes
 
 ## [v0.14.1](https://github.com/ava-labs/coreth/releases/tag/v0.14.1)
+
+- ⚠️ Archive node behavior change: by default, only the 24 hours preceeding the last accepted block are accepted for state proof queries. This can be adjusted with the new option `historical-proof-query-window` which defines the number of blocks before the last accepted block which should be accepted for state proof queries. This option can also be set to `0` to accept any block number state query.
```

Comment:
```suggestion
- IMPORTANT: `eth_getProof` calls for historical state will be rejected by default.
-- On archive nodes (`"pruning-enabled": false`): queries for historical proofs for state older than approximately 24 hours preceding the last accepted block will be rejected by default. This can be adjusted with the new option `historical-proof-query-window` which defines the number of blocks before the last accepted block which should be accepted for state proof queries, or set to `0` to accept any block number state query (previous behavior).
-- On `pruning` nodes: queries for proofs past the tip buffer (32 blocks) will be rejected. This is in support of moving to a path based storage scheme, which does not support historical state proofs.
```

Reply:
was this change undone on a force push?

Reply:
did we overwrite this suggestion with the old diff?

Reply:
Done in a65b60a97af5861fce1c81c24c35272aa2e6562d !
Just fixed the formatting to use ` -` instead of `--` which doesn't render fine (at least locally for me)